api/coreservice.go

```diff
@@ -127,18 +126,18 @@ type (
 		BlockMetas(start uint64, count uint64) ([]*iotextypes.BlockMeta, error)
 		// BlockMetaByHash returns blockmeta response by block hash
 		BlockMetaByHash(blkHash string) (*iotextypes.BlockMeta, error)
-		// LogsInBlock filter logs in the block x
-		LogsInBlock(filter *logfilter.LogFilter, blockNumber uint64) ([]*iotextypes.Log, error)
 		// LogsInBlockByHash filter logs in the block by hash
-		LogsInBlockByHash(filter *logfilter.LogFilter, blockHash hash.Hash256) ([]*iotextypes.Log, error)
+		LogsInBlockByHash(filter *logfilter.LogFilter, blockHash hash.Hash256) ([]*action.Log, error)
 		// LogsInRange filter logs among [start, end] blocks
-		LogsInRange(filter *logfilter.LogFilter, start, end, paginationSize uint64) ([]*iotextypes.Log, error)
+		LogsInRange(filter *logfilter.LogFilter, start, end, paginationSize uint64) ([]*action.Log, []hash.Hash256, error)
 		// EVMNetworkID returns the network id of evm
 		EVMNetworkID() uint32
 		// ChainID returns the chain id of evm
 		ChainID() uint32
 		// ReadContractStorage reads contract's storage
 		ReadContractStorage(ctx context.Context, addr address.Address, key []byte) ([]byte, error)
+		// RemoveListenerResponder deletes stream responder from chainlistener
+		RemoveListenerResponder(string) (bool, error)
```

Comment:
not used

Reply:
https://github.com/iotexproject/iotex-core/pull/3298/files#diff-6dc4e8a6418cb4f7e5b7da626b745fb04ff21d7c37b77d883b8c8b9ab7aacfcbR848